wrappers/dotnet/indy-sdk-dotnet/AnonCredsApi/AnonCreds.cs

```diff
@@ -225,14 +225,14 @@ private static void ProverGetCredentialsCallbackMethod(int xcommand_handle, int
 #if __IOS__
         [MonoPInvokeCallback(typeof(ProverSearchCredentialsCompletedDelegate))]
 #endif
-        private static void ProverSearchCredentialsCallbackMethod(int xcommand_handle, int err, IntPtr search_handle, int total_count)
+        private static void ProverSearchCredentialsCallbackMethod(int xcommand_handle, int err, int search_handle, int total_count)
         {
             var taskCompletionSource = PendingCommands.Remove<CredentialSearch>(xcommand_handle);
```

Comment:
I would like to see @srottem thoughts on changing these data types.  There is a slight difference between int and IntPtr that isn't obvious.